src/containers/Login.vue

```diff
@@ -6,20 +6,20 @@
       </div>
     </transition>
 
-    <div class="landing">
-      <!--TOP BAR-->
-      <top-bar />
-
+    <div class="landing top-sec">
       <!--PLAN AHEAD-->
       <div class="container p-0 m-0">
         <div class="row top-bar phonepad no-gutters">
           <div class="col">
             <div class="row no-gutters top-section">
-              <h1 class="plan-head mt-5">Plan Your Courses Ahead</h1>
+              <h1 class="plan-head mt-5">
+                Schedule Smarter <br />
+                Plan Ahead
+              </h1>
             </div>
             <div class="row no-gutters top-section">
               <p class="plan-subhead">
-                Introducing the new and easiest way to plan courses at Cornell
+                Introducing the newest and easiest way to plan courses at Cornell
```

Comment:
![image](https://user-images.githubusercontent.com/33106747/112039526-83b4b200-8b1a-11eb-9144-7f28d037d5f2.png)

It looks like this part is broken up into two lines and the font is a bit different on Figma